package.json

```diff
@@ -81,6 +81,8 @@
     "proton-shared": "github:ProtonMail/proton-shared.git#master",
     "react": "^16.8.1",
     "react-router-dom": "^4.3.1",
-    "ttag": "^1.7.0"
+    "ttag": "^1.7.0",
+    "pmcrypto": "github:ProtonMail/pmcrypto.git#semver:^6.0.0",
```

Comment:
I might be wrong but it comes with proton-pack, yeah only pm-srp is not inside. We should put them here as both are mandatory for all projects.